node-project-template/.eslintrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  // TODO: Uncomment this before commiting your new project
-  //"extends": "./node_modules/@usabilla/js-styleguide/eslint-config-usabilla/index.js"
```

Comment:
Removed this because it didn't seem necessary to keep it commented out. If there was some reason for it I'll put it back